spec/unit/puppet/provider/mysql_login_path/mysql_login_path_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
```

Comment:
Thanks for adding unit tests and acceptance tests. Will run them against release checks to verify on other OSes too. Thank you

Reply:
Thx